<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="acer-PC" name="Test retry twice" tests="4" failures="1" timestamp="3 Oct 2018 19:25:55 GMT" time="0.132" errors="0">
  <testcase name="test01" time="0.046" classname="fundamentals.RetryTesting">
    <skipped/>
  </testcase> <!-- test01 -->
  <testcase name="test01" time="0.003" classname="fundamentals.RetryTesting">
    <skipped/>
  </testcase> <!-- test01 -->
  <testcase name="test01" time="0.003" classname="fundamentals.RetryTesting">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at fundamentals.RetryTesting.test01(RetryTesting.java:11)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test01 -->
  <testcase name="test02" time="0.002" classname="fundamentals.RetryTesting"/>
</testsuite> <!-- Test retry twice -->
